syne_tune/experiments.py

```diff
@@ -46,6 +46,7 @@ class ExperimentResult:
     results: pd.DataFrame
     metadata: dict
     tuner: Tuner
+    path: Path
```

Comment:
Please also extend the docstring.

BTW: Is this a fix, or something that is needed for this example?

Reply:
It is needed for this example to get the path of experiment.